src/core/ext/load_reporting/load_reporting_filter.c

```diff
@@ -156,37 +158,36 @@ static void init_channel_elem(grpc_exec_ctx *exec_ctx,
                               grpc_channel_element *elem,
                               grpc_channel_element_args *args) {
   GPR_ASSERT(!args->is_last);
-
   channel_data *chand = elem->channel_data;
   memset(chand, 0, sizeof(channel_data));
 
-  chand->id = (intptr_t)args->channel_stack;
+  const grpc_auth_context *auth_context =
```

Comment:
Thanks! What's the advantage/difference wrt the current implementation? I see your suggestion does it per-call. Can the auth context or the peer identity change within the same channel?